### ISSUE TITLE
accessibility issue 9 (medium) - footer color contrast on links

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#239]
The footer link text on hover and on focus has an insufficient color contrast of 1.49:1.  The color contrast should be at least 4.5:1 at all times (including on hover and on focus).

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/a49a09fa4d4fcdb9d98e6941a5134c78768d8aa2

also added to footer bar Yale icon the href=https://www.yale.edu
